f8a_report/helpers/report_helper.py

```diff
@@ -90,21 +90,21 @@ def cleanup_db_tables(self):
             num_days = os.environ.get('KEEP_WORKER_RESULT_NUM_DAYS', '30')
             self.cleanup_tables('worker_results', 'ended_at', num_days)
 
-            # Number of days to retain the package worker results data
-            num_days = os.environ.get('KEEP_PACKAGE_WORKER_RESULT_NUM_DAYS', '30')
-            self.cleanup_tables('package_worker_results', 'ended_at', num_days)
-
-            # Number of days to retain the package analyses data
-            num_days = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '30')
-            self.cleanup_tables('package_analyses', 'finished_at', num_days)
-
             # Number of days to retain the stack analyses request data
             num_days = os.environ.get('KEEP_STACK_ANALYSES_REQUESTS_NUM_DAYS', '180')
             self.cleanup_tables('stack_analyses_request', 'submitTime', num_days)
 
             # Number of days to retain the api requests data
             num_days = os.environ.get('KEEP_API_REQUESTS_NUM_DAYS', '180')
             self.cleanup_tables('api_requests', 'submit_time', num_days)
+
+            # Number of days to retain the package analyses data
+            num_days = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '31')
```

Comment:
This order will not work, foreign key needs to be deleted first.

Reply:
Resolved.